src/View/Includes/Banner/pages/_bannerExemple.html.twig

```diff
@@ -0,0 +1,8 @@
+{% extends "/Includes/Banner/layoutBanner.html.twig" %}
+
+{% block picture %}
+    <img class="img-banner" src="/assets/images/collector.webp" alt="mascotte">
+{% endblock %}
+{% block pageTitle %}
+    <h1 class="text-light">Haras du Val de Loire</h1>
+{% endblock %}
```

Comment:
this extends a component? +these blocks seem

Reply:
correction done